src/courses/forms.py

```diff
@@ -117,16 +117,31 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        self.fields['name'].help_text = 'If empty, will instead use First day as it\'s name'
+        self.fields['name'].required = False
```

Comment:
This should also just get picked up from the model field's `required=False` parameter

Reply:
changed